scripts/test-playwright.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# npm run test-playwright --config="jest.ui.config.js"
+
+npx jest --config=jest.ui.config.js
+test_result=$?
+
+exit $test_result
```

Comment:
Is it possible to reduce complexity and remove this sh script?

Reply:
Would you prefer both scripts inlined? Would maybe be worth a thought.

Reply:
Dont like these shellscripts. Isnt it possible to just start "npx jest..." instead of the sh file?

Reply:
Yeah this is what I mean. To be consistent, we would need to do the same for the testcafe-script, not just playwright. What do you think?